addons/binding/org.openhab.binding.gpio/README.md

```diff
@@ -0,0 +1,12 @@
+# GPIO #
+## Introduction ##
+This binding adds GPIO support via the pi4j libary for openhab2.
+## Features ##
+
+ - GPIO switch: turn GPIO output on and off
+ - GPIO contact: get state of GPIO pin
+
+## How to use ##
+You have to install [wiringpi](http://wiringpi.com/) and run openhab as root (sudo, etc..) to use the addon. 
```

Comment:
It would be so much easier for users if you bundled the native library with the binding using OSGi's Bundle-NativeCode instruction, and allowed use by non-root users that belong to the appropriate user group.  So installation of wiringpi would not be needed and openHAB could run as installed normally.  Otherwise, it's excellent to be using Pi4J for this binding!